### PR TITLE
Add ItemCollection fragment #25

### PR DESCRIPTION
**Pull request by m-mohr**
**Related Issue(s):** #25

**Proposed Changes:**

1. Add ItemCollection fragment back in

**PR Checklist:**

- [x] This PR is made against the dev branch (all proposed changes except releases should be against dev, not master).
- [x] This PR has **no** breaking changes.
- [ ] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-api-spec/blob/dev/CHANGELOG.md) **or** a CHANGELOG entry is not required.
- [ ] OpenAPI files only: I have run `npm run generate-all` to update the [generated OpenAPI files](https://github.com/radiantearth/stac-api-spec/blob/dev/README.md#openapi-definitions).

### CODE REVIEW

File: core/examples/itemcollection-sample-full.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
+  "stac_version": "0.9.0",
```
Comment (m-mohr) [inline core/examples/itemcollection-sample-full.json:2]:
```suggestion
  "stac_version": "1.0.0-beta.2",
```

File: core/examples/itemcollection-sample-full.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  "stac_extensions": [],
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "stac_version": "0.9.0",
```
Comment (m-mohr) [inline core/examples/itemcollection-sample-full.json:7]:
```suggestion
      "stac_version": "1.0.0-beta.2",
```

File: core/examples/itemcollection-sample-minimal.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "stac_version": "0.9.0",
```
Comment (m-mohr) [inline core/examples/itemcollection-sample-minimal.json:2]:
```suggestion
  "stac_version": "1.0.0-beta.2",
```

File: core/itemcollection-spec.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# STAC ItemCollection Specification
```
Comment (m-mohr) [inline core/itemcollection-spec.md:1]:
```suggestion
# STAC API - ItemCollection Fragment
```

+ rename file to README.md

File: core/itemcollection-spec.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Items are represented in JSON format and are very flexible. Any JSON object that contains all the
+required fields is a valid STAC ItemCollection.
+
+- Examples:
+  - See the [minimal example](examples/itemcollection-sample-minimal.json), as well as a [more complete 
+    example](examples/itemcollection-sample-full.json). There are more real world inspired samples in the [examples/](examples/) folder.
+- [OpenAPI YAML](openapi.yaml)
+
+## WARNING
```
Comment (m-mohr) [inline core/itemcollection-spec.md:23]: Remove the warning?!

Comment (cholmes) [reply core/itemcollection-spec.md]: Yes :)

**Review by cholmes — APPROVED** [review]

**Pull request merged**